src/mbgl/tile/raster_dem_tile.cpp

```diff
@@ -20,6 +20,16 @@ RasterDEMTile::RasterDEMTile(const OverscaledTileID& id_,
       mailbox(std::make_shared<Mailbox>(*Scheduler::GetCurrent())),
       worker(parameters.workerScheduler,
              ActorRef<RasterDEMTile>(*this, mailbox)) {
+
+    if ( id.canonical.y == 0 ){
+        // this tile doesn't have upper neighboring tiles so marked those as backfilled
+        neighboringTiles = neighboringTiles | 0b00011100;
```

Comment:
Would be safer to use the enum values here. Or create an enum value for this constant.